### PR TITLE
Refactor/update custom table accessibility

### PR DESCRIPTION
**Pull request by GuillaumeKESTEMAN**
Update shared table accessibility : 
- add scope attributes
- update/remove some aria labels
- add required caption with screen reader visibility management ([see tailwind sr-only](https://tailwindcss.com/docs/screen-readers))

Other things :
- add imports organization on file save
- replace snapshots by manual tests (on files updated by previous changes)

- [ ] Update npm packages
- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Performance improvement
- [X] Refactoring (so no functional change)
- [ ] Other (docs, build config, etc)

## Checklist:

- [X] I've added tests for my code. (most of the time mandatory)
- [ ] I have added an entry to the changelog. (mandatory)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx [hunk 1]
```diff
@@ -6,20 +6,21 @@ const defaultProps = {
...
-    const { asFragment } = renderWithContainer(
+    const { container } = renderWithContainer(
       <Td {...defaultProps} classModifier="custom" hover={<p>Hover content</p>}>
         child td
       </Td>,
-      container,
+      trContainer,
     );
-    expect(asFragment()).toMatchSnapshot();
+    expect(container.firstChild?.firstChild).toHaveClass('af-popover__wrapper');
```
Comment (GuillaumeKESTEMAN) [inline src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx:24]: J'ai un doute ici s'il ne faut pas vérifier que `container.firstChild?.firstChild` existe car ça peut planter s'il n'existe pas non ?

File: src/shared/components/Table/Body/__tests__/Body.test.tsx [hunk 2]
```diff
@@ -23,6 +24,10 @@ describe('Body', () => {
 
   it('Render <Body/> with 1 item', () => {
     const { baseElement } = renderWithContainer(<Body items={items} />, container);
-    expect(baseElement).toMatchSnapshot();
+    expect(screen.getAllByRole('row').length).toBe(1);
```
Comment (GuillaumeKESTEMAN) [inline src/shared/components/Table/Body/__tests__/Body.test.tsx:27]:
```suggestion
    expect(within(baseElement).getAllByRole('row').length).toBe(1);
```

File: src/shared/components/Table/Header/__tests__/Header.test.tsx [hunk 3]
```diff
@@ -29,12 +30,29 @@ describe('Header', () => {
...
+    let columnHeaderNumber = headers !== undefined ? headers.length : 0;
+    // when sorting, the th tag takes a 'button' role and is therefore not a columnheader
+    if (sortingMock && columnHeaderNumber > 0) {
+      columnHeaderNumber -= 1;
+    }
+    if (children) {
+      columnHeaderNumber += 1;
+    }
+
+    expect(screen.queryAllByRole('columnheader').length).toBe(columnHeaderNumber);
```
Comment (GuillaumeKESTEMAN) [inline src/shared/components/Table/Header/__tests__/Header.test.tsx:50]:
```suggestion
    expect(within(baseElement).queryAllByRole('columnheader').length).toBe(columnHeaderNumber);
```

File: src/shared/components/Table/Header/__tests__/Header.test.tsx [hunk 4]
```diff
@@ -29,12 +30,29 @@ describe('Header', () => {
     ${undefined}   | ${sortingMock} | ${undefined}
     ${headersMock} | ${sortingMock} | ${(<th>child header</th>)}
   `('Should render Header when headers: $headers, sorting: $sorting', ({ headers, sorting, children }) => {
-    const { baseElement } = renderWithContainer(
+    renderWithContainer(
```
Comment (GuillaumeKESTEMAN) [inline src/shared/components/Table/Header/__tests__/Header.test.tsx:33]:
```suggestion
   const { baseElement } = renderWithContainer(
```

File: src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx [hunk 5]
```diff
@@ -6,20 +6,21 @@ const defaultProps = {
...
 };
-const container = document.createElement('tr');
+const trContainer = document.createElement('tr');
 
 describe('Td', () => {
   it('Render <Td/> without hover', () => {
-    const { asFragment } = renderWithContainer(<Td {...defaultProps}>child th</Td>, container);
-    expect(asFragment()).toMatchSnapshot();
+    const { container } = renderWithContainer(<Td {...defaultProps}>child th</Td>, trContainer);
+    expect(container.firstChild?.firstChild?.textContent).toBe('label');
```
Comment (samuel-gomez) [inline src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx:14]: ici tu pouvais simplement faire un getByText

File: src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx (hunk 1 above)
Comment (samuel-gomez) [reply src/shared/components/Table/Body/Line/Td/__tests__/Td.test.tsx:24]: idem getByText

File: src/pages/Demos/Members/Members.tsx [hunk 6]
```diff
@@ -1,11 +1,11 @@
 import { Paging } from '@axa-fr/react-toolkit-all';
-import Resilience from 'shared/components/Resilience';
 import Layout, { type TLayoutPage } from 'Layout';
-import { Tanomaly } from 'shared/types';
 import Loader, { type TLoader } from 'shared/components/Loader';
+import Resilience from 'shared/components/Resilience';
 import Table from 'shared/components/Table';
-import { TITLE_BAR, TITLE, TABLE_HEADERS_MEMBERS } from './constants';
+import { Tanomaly } from 'shared/types';
```
Comment (samuel-gomez) [inline src/pages/Demos/Members/Members.tsx:6]: import type

File: src/shared/components/Table/Header/Th/ThSortable/__tests__/ThSortable.test.tsx [hunk 7]
```diff
@@ -17,15 +19,19 @@ describe('ThSortable', () => {
...
+      const { baseElement, container } = renderWithContainer(
+        <ThSortable {...defaultProps} {...rest} order={order}>
           {children}
         </ThSortable>,
-        container,
+        trContainer,
       );
 
-      expect(baseElement).toMatchSnapshot();
+      expect(container.querySelector('span')).toHaveClass(`glyphicon-${orderIcons(order)}`);
```
Comment (samuel-gomez) [inline src/shared/components/Table/Header/Th/ThSortable/__tests__/ThSortable.test.tsx:30]: tu pouvais juste faire un querySelector qui vise la class et vérifier qu'il est dans le document

**Review by samuel-gomez — APPROVED** [review]

**Pull request merged**